adlarge/fixtures-documentation-bundle/1.1/manifest.json

```diff
@@ -0,0 +1,11 @@
+{
+    "bundles": {
+        "Adlarge\\FixturesDocumentationBundle\\AdlargeFixturesDocumentationBundle": ["dev"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    },
+    "composer-scripts": {
+        "assets:install --symlink --relative %PUBLIC_DIR%": "symfony-cmd"
```

Comment:
This is related to the assets package. You dont need to include this here.

Reply:
Thank you, i removed it

Reply:
There is an issue with ci now because of php 7.4, can you merge it ? or do i have to limit php version ?

Reply:
Thank you. 
I will merge this. 
When making a PR to the recipes one should never change the CI config =)